en/sql/i18n.rst

```diff
@@ -1816,21 +1816,32 @@ Linux
 Timezone library and database compatibility
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The timezone library build by CUBRID includes an MD5 checksum of the timezone data it contains. This hash is stored in all databases which are created with that library, into column **timezone_checksum** of **db_root** system table. If timezone library is changed (recompiled with newer timezone data) and the checksum changes, the CUBRID server, nor any other CUBRID tool cannot be started with existing databases.
+The timezone library built by CUBRID includes an MD5 checksum of the timezone data it contains. This hash is stored in all databases which are created with that library, into column **timezone_checksum** of **db_root** system table. If timezone library is changed (recompiled with newer timezone data) and the checksum changes, the CUBRID server, nor any other CUBRID tool cannot be started with existing databases.
 To avoid such incompatibility, one option is to use /extend argument of the make_tz tool. When this option is provided, the **timezone_checksum** value of database is also changed with the new MD5 checksum of the new timezone library.
+The extend feature should be used when you decide to use a different version of timezone library from the IANA site. It does two things:
+
+-   It generates a new library by merging the old timezone data with the new timezone data. 
+    After the merge, all the timezone regions that aren't present in the new timezone database are kept in order to ensure backward compatibility 
+    with the data in the database tables.
+ 
+-   The second thing that it does is to update the timezone data in the tables in the situation when backward compatibility 
+    could not be ensured in the first phase when the new timezone library was generated. This situation can occur 
+    when an offset rule or a daylight saving rule changes.
+  
+    When you run make_tz with the extend option all the databases on your machine are updated together with the MD5 checksum.
```

Comment:
`all the databases on you machine` may confuse users. 
`all the databases on your database directory file(databases.txt)` might be better. 

I'd like to also explicitly explain the current restrictions. For instance, multiple users share one CUBRID installation copy should migrate all of their databases at once.